src/tiles/tile-group.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	Component,
+	AfterContentInit,
+	Input,
+	Output,
+	EventEmitter,
+	HostBinding,
+	ContentChildren,
+	QueryList
+} from "@angular/core";
+import { SelectionTile } from "./selection-tile.component";
+import { NG_VALUE_ACCESSOR } from "@angular/forms";
+
+@Component({
+	selector: "ibm-tile-group",
+	template: `<ng-content select="ibm-selection-tile"></ng-content>`,
+	providers: [
+		{
+			provide: NG_VALUE_ACCESSOR,
+			useExisting: TileGroup,
+			multi: true
+		}
+	]
+})
+export class TileGroup implements AfterContentInit {
+	static tileGroupCount = 0;
+	/**
+	 * The tile group `name`
+	 */
+	@Input() name = `tile-group-${TileGroup.tileGroupCount}`;
+	/**
+	 * Set to `true` to support multiple tile selection
+	 */
+	@Input() multiple = false;
+
+	/**
+	 * Emits an event when the tile selection changes.
+	 *
+	 * Emits an object that looks like:
+	 * ```javascript
+	 * {
+	 * 	value: "something",
+	 * 	selected: true,
+	 * 	name: "tile-group-1"
+	 * }
+	 * ```
+	 */
+	@Output() tileSelection: EventEmitter<{value: string, selected: boolean, name: string}> = new EventEmitter();
```

Comment:
Same comment as before, can we encode `{value: string, selected: boolean, name: string}` as an explicit type?